File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT1101Q01PrintNameInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroOUT1101Q01PrintNameInfo
    {
        public NuroOUT1101Q01PrintNameInfo() {}
    
        private string _codeName = "";
        public string CodeName
        {
            get { return _codeName; }
            set { _codeName = value; }
        }
        private string _sortKey = "";
        public string SortKey
        {
            get { return _sortKey; }
            set { _sortKey = value; }
        }
    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT1101Q01PrintNameInfo.java
package nta.med.data.model.ihis.nuro;

import java.io.Serializable;

public class NuroOUT1101Q01PrintNameInfo implements Serializable{
	private String codeName;
    private String sortKey;
	public NuroOUT1101Q01PrintNameInfo(String codeName, String sortKey) {
		super();
		this.codeName = codeName;
		this.sortKey = sortKey;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getSortKey() {
		return sortKey;
	}
	public void setSortKey(String sortKey) {
		this.sortKey = sortKey;
	}
    
}
